www/include/Administration/brokerPerformance/brokerPerformance.ihtml

```diff
@@ -18,7 +18,13 @@
     {foreach from=$perf_info_array item=perf_info key=conf_name}
     <tr>
         <td class='list_lvl_1' colspan='2'>
-            <p>{$conf_name}&nbsp;<span class="broker_stats_lastmodif">{$lang.updated}:&nbsp;{$perf_info.lastmodif}</span></p>
+            <p>{$conf_name}&nbsp;
+
+                <!-- uniformisation date -->
```

Comment:
Good thing to comment, but use english :)

Reply:
change done